orchagent/muxorch.cpp

```diff
@@ -1183,13 +1184,13 @@ void MuxOrch::update(SubjectType type, void *cntx)
     }
 }
 
-MuxOrch::MuxOrch(DBConnector *db, const std::vector<std::string> &tables,
+MuxOrch::MuxOrch(DBConnector *cfg_db, DBConnector *app_db, const std::vector<std::string> &tables,
          TunnelDecapOrch* decapOrch, NeighOrch* neighOrch, FdbOrch* fdbOrch) :
-         Orch2(db, tables, request_),
+         Orch2(cfg_db, tables, request_),
          decap_orch_(decapOrch),
          neigh_orch_(neighOrch),
          fdb_orch_(fdbOrch),
-         cfgTunnelTable_(db, CFG_TUNNEL_TABLE_NAME)
+         app_decap_tunnel_table_(app_db, APP_TUNNEL_DECAP_TABLE_NAME)
```

Comment:
I think there is a gap here. Let us discuss offline

Reply:
Fixed as discussion. Please help take a look. Thanks